docs/installation/autoware/docker-installation.md

```diff
@@ -1,106 +1,161 @@
-...
-
-Collecting staticx==0.12.3
-Downloading https://files.pythonhosted.org/packages/92/ff/d9960ea1f9db48d6044a24ee0f3d78d07bcaddf96eb0c0e8806f941fb7d3/staticx-0.12.3.tar.gz (68kB)
-Complete output from command python setup.py egg_info:
-Traceback (most recent call last):
-File "", line 1, in
-File "/tmp/pip-install-m_nm8mya/staticx/setup.py", line 4, in
-from wheel.bdist_wheel import bdist_wheel
-ModuleNotFoundError: No module named 'wheel'
-
-Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-m_nm8mya/staticx/
-...
+./docker/run.sh --devel
 ```
 
-This error exists in current version of rocker tool, which relates to the os_detection function of rocker.
+!!! info
 
-To fix this error, temporary modification of rocker source code is required, which is not recommended.
+    By default workspace mounted on the container will be current directory, you can change the workspace path by `--workspace path_to_workspace`. For development environments without NVIDIA GPU support use `--no-nvidia`.
 
-At current stage, it is recommended to run docker without NVIDIA gpu enabled for arm64 devices:
+#### How to Setup a Workspace
 
-```bash
-rocker -e LIBGL_ALWAYS_SOFTWARE=1 --x11 --user --volume $HOME/autoware -- ghcr.io/autowarefoundation/autoware-universe:latest-cuda
+1. Create the `src` directory and clone repositories into it.
+
+   ```bash
+   mkdir src
+   vcs import src < autoware.repos
+   ```
+
+2. Update dependent ROS packages.
+
+   The dependency of Autoware may change after the Docker image was created.
+   In that case, you need to run the following commands to update the dependency.
+
+   ```bash
+   sudo apt update
+   rosdep update
+   rosdep install -y --from-paths src --ignore-src --rosdistro $ROS_DISTRO
+   ```
+
+3. Build the workspace.
+
+   ```bash
+   colcon build --symlink-install --cmake-args -DCMAKE_BUILD_TYPE=Release
+   ```
+
+   If there is any build issue, refer to [Troubleshooting](../../support/troubleshooting/index.md#build-issues).
+
+> **To Update the Workspace**
+>
+> ```bash
+> cd autoware
+> git pull
+> vcs import src < autoware.repos
+> vcs pull src
+> ```
+
+#### Using VS Code Remote Containers for Development
+
+Using the [Visual Studio Code](https://code.visualstudio.com/) with the [Remote - Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) extension, you can develop Autoware in the containerized environment with ease.
+
+Get the Visual Studio Code's [Remote - Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) extension.
+And reopen the workspace in the container by selecting `Remote-Containers: Reopen in Container` from the Command Palette (`F1`).
+
+By default devcontainer assumes NIVIDA GPU support, you can change this by deleting these lines within `.devcontainer/devcontainer.json`:
```

Comment:
```suggestion
By default devcontainer assumes NVIDIA GPU support, you can change this by deleting these lines within `.devcontainer/devcontainer.json`:
```

Reply:
fixed in https://github.com/autowarefoundation/autoware-documentation/pull/516/commits/2e004565db59712ca7628cc6d07d50bf9f5f94d6